src/components/CommentList/CommentList.scss

```diff
@@ -1 +1,7 @@
-//
+.CommentList {
+  display: flex;
+  flex-direction: column;
+  gap: 0.7em;
+  background-color: #eee;
+  padding: 1em;
+}
\ No newline at end of file
```

Comment:
Add a newline at the end of the file for better readability and to follow best practices